spec/models/accounting/quickbooks/department_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
```

Comment:
There are _slight_ differences with Company. This one has a `name` not a `display_name`. It also returns a `BaseReference` instead of an entity. Any suggestions on how dry up the differences, or if I should?